index.html

```diff
@@ -0,0 +1,162 @@
+      </div>
+    </header>
+    <section class="intro">
+      <nav class="header_menu">
+        <ul class="Desk_nav">
+          <li class="Desk_list"><a href="./about.html"> About </a></li>
+          <li class="Desk_list"><a href="#work"> Program </a></li>
+          <li class="Desk_list"><a href="#contact-form"> Join </a></li>
+          <li class="Desk_list"><a href="#work"> Sponsor </a></li>
+          <li class="Desk_list"><a href="#contact-form"> News </a></li>
+        </ul>
+      </nav>
+      <div class="humburger" d>
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="introContainer">
+        <p>"Hello! Great People"</p>
+        <h1>
+          Timbuktu Technical <br />
+          Institute Calling<br />
+          2023
+        </h1>
+        <p class="para3">
+          Timbuktu Technical Institute is calling for people all over the
+          Beautiful African Continent to apply to the free programs it offers.
+          Register on this site for free.
+        </p>
+        <h2>23.02.2023(Thurs) ~ 27(Mon)</h2>
+        <h3>@ Timbuktu Technical Institute offices across the continent</h3>
+      </div>
+    </section>
+    <section class="main">
+      <h1>Main Programs</h1>
+      <hr>
+      <div class="mainActivities">
+        <div class="activity">
+          <img src="./Images/Lecture.png" alt="Lecture Image" />
+          <h2>Lecture</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <div class="activity">
+          <img src="./Images/exhibition.png" alt="Exhibition Image" />
+          <h2>TTI exhibition</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <div class="activity">
+          <img src="./Images/forum.jpg" alt="Forum Image" />
+          <h2>TTI Forum</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <div class="activity">
+          <img src="./Images/workshop.png" alt="Workshop Image" />
+          <h2>Workshop</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <div class="activity">
+          <img src="./Images/networking.webp" alt="networking Image" />
+          <h2>Network</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <h3 class="join_program_link">Join TTI Program 2023</h3>
```

Comment:
- [ ] This line should be `centered` always for the mobile version according to the given design. Kindly fix this issue.
  - Your design
  - ![Screenshot from 2023-02-09 22-17-53](https://user-images.githubusercontent.com/60083437/217872403-b6f978fb-c907-482f-94e7-e5d1d5554b2f.png)
  - Given design
  - ![Screenshot from 2023-02-09 22-21-23](https://user-images.githubusercontent.com/60083437/217872517-ae255ba4-7f2f-45fc-9293-33baac87394f.png)

Reply:
Okay, I will ensured that the text is centered.